package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-cpptools",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Comment:
Shouldn't this be 3.2.0? I thought major upgrades were supposed to be for breaking changes. A lower version would allow for more consumers to auto-update.

Reply:
Oh, it might be a breaking for consumers of the API if they don't expect the new enum values to be passed, so 4.0.0 seems fine.